docs/source/models/supported_models.rst

```diff
@@ -168,3 +168,49 @@ Alternatively, you can raise an issue on our `GitHub <https://github.com/vllm-pr
         llm = LLM(model=..., revision=..., trust_remote_code=True)  # Name or path of your model
         output = llm.generate("Hello, my name is")
         print(output)
+
+Model Support Policy
+---------------------
+
+At vLLM, we are committed to facilitating the integration and support of third-party models within our ecosystem. Our approach is designed to balance the need for robustness and the practical limitations of supporting a wide range of models. Here’s how we manage third-party model support:
+
+1. **Community-Driven Support**: We encourage community contributions for adding new models. When a user requests support for a new model, we welcome pull requests (PRs) from the community. These contributions are evaluated primarily on the sensibility of the output they generate, rather than strict consistency with existing implementations such as those in transformers.
```

Comment:
Probably also worth pointing out that a basic sensibility test is also required in the model support PR.

Reply:
A sensibility report is required in the PR. However, with respect to adding it into test, there is a strong concern on the time and resource burden it adds to our CI system.

Reply:
Totally make sense - I think an attached report from the author in the PR is fine for now and we don't need to build it into our CI.